2.0/runtime/nanoserver-insider/README.md

```diff
@@ -0,0 +1,163 @@
+
+You need to create a `Dockerfile` with the following:
+
+```dockerfile
+FROM microsoft/nanoserver-insider-dotnet:sdk AS build-env
+WORKDIR /dotnetapp
+
+# copy csproj and restore as distinct layers
+COPY *.csproj ./
+RUN dotnet restore
+
+# copy everything else and build
+COPY . ./
+RUN dotnet publish -c Release -o out
+
+# build runtime image
+FROM microsoft/nanoserver-insider-dotnet:runtime 
+WORKDIR /dotnetapp
+COPY --from=build-env /dotnetapp/out ./
+ENTRYPOINT ["dotnet", "dotnetapp.dll"]
+```
+
+Build and run the Docker image:
+
+```console
+docker build -t dotnetapp .
+docker run -it --rm dotnetapp
+```
+
+The `Dockerfile` and the Docker commands assumes that your application is called `dotnetapp`. You can change the `Dockerfile` and the commands, as needed.
+
+## Interactively build and run a simple .NET Core application
+
+You can interactively try out .NET Core by taking advantage of the convenience of a container. Try the following set of commands to create and run a .NET Core application in a minute (depending on your internet speed).
+
+```console
+docker run -it --rm microsoft/nanoserver-insider-dotnet
+[now in the container]
+mkdir app
+cd app
+dotnet new console
+dir
+dotnet restore -s https://dotnet.myget.org/F/dotnet-core/api/v3/index.json -s https://api.nuget.org/v3/index.json
+dotnet run
+dotnet bin/Debug/netcoreapp2.0/app.dll
+dotnet publish -c Release -o out
+dotnet out/app.dll
+exit
+ ```
+
+The steps above are intended to show the basic functions of .NET Core tools. Try running `dotnet run` twice. You'll see that the second invocation skips compilation. The subsequent command after `dotnet run` demonstrates that you can run an application directly out of the bin folder, without the additional build logic that `dotnet run` adds. The last two commands demonstrate the publishing scenario, which prepares an app to be deployed on the same or other machine, with a requirement on only the .NET Core Runtime, not the larger SDK. Naturally, you don't have to exit immediately, but can continue to try out the product as long as you want.
+
+## Image variants
+
+The `microsoft/nanoserver-insider-dotnet` images come in different flavors, each designed for a specific use case.
+
+### `microsoft/nanoserver-insider-dotnet:<version>-sdk`
+
+This is the defacto image. If you are unsure about what your needs are, you probably want to use this one. It is designed to be used both as a throw away container (mount your source code and start the container to start your app), as well as the base to build other images off of.
+
+It contains the .NET Core SDK which is comprised of two parts:
+
+1. .NET Core
+2. .NET Core command line tools
+
+Use this image for your development process (developing, building and testing applications).
+
+### `microsoft/nanoserver-insider-dotnet:<version>-runtime`
+
+This image contains the .NET Core (runtime and libraries) and is optimized for running .NET Core apps in production.
```

Comment:
Parenthetical is a little awkward, not sure if any info is lost to just say "...contains the .NET Core runtime and libraries...".